### ISSUE TITLE
Rerun Signature Module with All TARGET samples

### ISSUE THREAD

**Issue by gwaygenomics** [greenelab/BioBombe#187]
In #186 I update the t-test analysis to visualize the test set in TCGA and GTEx. I am currently using the training set in TARGET because of low sample sizes. Let's update this to _both_ the training set and test set for the same reason.
